server/services/ebayScraperService.js

```diff
@@ -0,0 +1,47 @@
+const puppeteer = require("puppeteer");
+
+exports.getProductDetail = async (productUrl) => {
+  if (productUrl) {
+    try {
+      const browser = await puppeteer.launch({ headless: true });
+      const page = await browser.newPage();
+      await page.goto(productUrl, { waitUntil: "domcontentloaded" });
+
+      const productData = await page.evaluate(() => {
+        //product description
+        // uncomment if we decide to include post description later
+        // const postBody = document.querySelector("#postingbody").innerText.replaceAll('\n', ' ');
+
+        //product price
+        let price = "unavailable";
+
+        try {
+          price = document.getElementById("prcIsum").innerText;
```

Comment:
u can also use the contetn atribute to get price without the $ sign
`document.getElementById("prcIsum").getAttribute('content')` I tested this on chrome dev tools